charts/argocd-notifications/templates/deployment.yaml

```
@@ -29,6 +29,14 @@ spec:
           command:
             - /app/argocd-notifications
             - controller
+            - --loglevel={{ .Values.logLevel }}
+            {{- if .Values.metrics.enabled }}
+            - --metrics-port={{ .Values.metrics.port }}
+          ports:
+          - containerPort: {{ .Values.metrics.port }}
+            name: metrics
+            protocol: TCP
+            {{- end }}
```

I think I understand what is going on here though it took me a double take.  How would you feel about separating out the conditional between the command and ports?
```suggestion
            {{- if .Values.metrics.enabled }}
            - --metrics-port={{ .Values.metrics.port }}
            {{- end }}
          ports:
          {{- if .Values.metrics.enabled }}
          - containerPort: {{ .Values.metrics.port }}
            name: metrics
            protocol: TCP
          {{- end }}
```

---

updated, thanks

---

apologies but not seeing the updates.  as soon as I do, I will get them reviewed and shipped

---

@andyfeller this thread is `Outdated`, do you see the changes in `FilesChanged`?